gio/Gir.toml

```diff
@@ -310,6 +310,14 @@ generate_builder = true
     name = "open"
     manual = true
     doc_trait_name = "ApplicationExtManual"
+    [[object.signal]]
+    name = "command-line"
+    manual = true
+    doc_trait_name = "ApplicationExtManual"
+    [[object.signal]]
+    name = "handle-local-options"
+    manual = true
+    doc_trait_name = "ApplicationExtManual"
```

Comment:
i don't think the doc_trait_name is needed anymore

Reply:
I just followed the style of the `[[object.signal]]` override for the "open" just above these. Should I remove the `doc_trait_name` for that one too?

Reply:
I think it's still used to generate the documentation on the `connect_...` functions in `ApplicationExtManual`. It's still used here: https://github.com/gtk-rs/gir/blob/491114ad76bde3d0c2c30312638782c6638c00c2/src/codegen/doc/mod.rs#L537-L561